testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -167,7 +167,27 @@ def test_write_frame_none(self, u, tmpdir, extension):
             u.atoms.positions[None, ...], new_positions, decimal=2
         )
 
+    @pytest.mark.parametrize('extension', ('xtc', 'dcd', 'pdb', 'xyz'))
```

Comment:
Se should capitalized extensions, too.

Reply:
I'm afraid I don't understand. Should I capitalize all the extensions?. I've used the parameters from the previous test already written.

Reply:
I've added a test for a capitalized extension 'PDB'.

Reply:
Sorry, clearly I should proof-read what I am typing... yes, test for capitalized extensions was what I was looking for.